src/data/2_board.json

```diff
@@ -0,0 +1,97 @@
+{
+    "board": {
+        "blueturns": [
+            {
+                "guess": "ab",
+                "score": 100
+            },
+            {
+                "guess": "cde",
+                "score": 200
+            },
+            {
+                "guess": "fgh",
+                "score": 300
+            },            
+            {
+                "guess": "ijk",
+                "score": 400
+            },
+            {
+                "guess": "lmn",
+                "score": 500
+            },
+            {
+                "guess": "opqr",
+                "score": 600
+            },
+            {
+                "guess": "stuv",
+                "score": 700
+            },
+            {
+                "guess": "wxyz",
+                "score": 800
+            },
+            {
+                "guess": "aaaaa",
+                "score": 900
+            },
+            {
+                "guess": "bbbbb",
+                "score": 1000
+            },
+            {
+                "guess": "cccccc",
+                "score": 1100
+            }
+        ],
+        "beigeturns": [            
+            {
+                "guess": "ab",
+                "score": 2100
+            },
+            {
+                "guess": "cde",
+                "score": 2200
+            },
+            {
+                "guess": "fgh",
+                "score": 2300
+            },            
+            {
+                "guess": "ijk",
+                "score": 2400
+            },
+            {
+                "guess": "lmn",
+                "score": 2500
+            },
+            {
+                "guess": "opqr",
+                "score": 2600
+            },
+            {
+                "guess": "stuv",
+                "score": 2700
+            },
+            {
+                "guess": "wxyz",
+                "score": 2800
+            },
+            {
+                "guess": "aaaaa",
+                "score": 2900
+            },
+            {
+                "guess": "bbbbb",
+                "score": 3000
+            },
+            {
+                "guess": "cccccc",
+                "score": 3100
+            }
+        ],
+        "blueword": "HIDDEN"
```

Comment:
I like these name changes. I think they should be camel case per JSON conventions, though, e.g. `blueWord`